src/js/polyfills.js

```diff
@@ -12,3 +12,746 @@ if (!Element.prototype.matches) {
   Element.prototype.matches = Element.prototype.msMatchesSelector ||
     Element.prototype.webkitMatchesSelector;
 }
+
```

Comment:
My thought is let's yank the polyfill. All modern browsers support Intersection Observer. This feature isn't crucial for people enough for it to support IE11 -- it's more of a "nice to have" IMHO

Reply:
Works for me - this polyfill is a beast.

Reply:
Pushed an update that removes the polyfill.